ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivacyController.java

```diff
@@ -173,4 +178,38 @@ public long getSenderNonce(final Address sender, final String privacyGroupId) {
             // private state does not exist
             Account.DEFAULT_NONCE);
   }
+
+  private SendResponse sendRequest(final PrivateTransaction privateTransaction) {
+    final BytesValueRLPOutput bvrlp = new BytesValueRLPOutput();
```

Comment:
should just be an RlpOutput object (newed as a BytesValueRlpOutput) - and probalby needs a nicer name - even its its just rlpOutput